### PR TITLE
clean up community doc to removec outdated links

### PR DESCRIPTION
**Pull request by hbagdi**
- removes the outdated cschedule scratchpad
- removes the old design doc

### CODE REVIEW

File: docs/community/index.html [hunk 1]
```diff
@@ -511,89 +497,27 @@ <h3 id="office-hours">Office Hours</h3>
...
-<td>Sketch of the proposed API</td>
-</tr>
-</tbody>
-</table>
+<p>Meeting agendas and notes are recorded in the <a href="https://docs.google.com/document/d/1eg-YjOHaQ7UD28htdNxBR3zufebozXKyI28cl2E11tU/edit">meeting notes doc</a>.</p>
+<p>All meetings are recorded and automatically uploaded to youtube:
+<a href="https://www.youtube.com/playlist?list=PL69nYSiGNLP2E8vmnqo5MwPOY25sDWIxb">Meeting recordings</a>.</p>
+<p>Some initial recordings of this working group were done manually and can be
+found in the below table:
+| Date               |                                |
```
Comment (robscott) [inline docs/community/index.html:505]: Looks like the table generation got broken here - maybe a line break above this would help?

Comment (hbagdi) [reply docs/community/index.html]: Whoops, fixed. Good catch.

**Comment by hbagdi** [comment]
@robscott PTAL

**Comment by robscott** [comment]
Thanks!

/lgtm
/approve

**Pull request merged**